build-system-tests/package.json

```diff
@@ -23,22 +23,54 @@
     "setup:react-native": "./scripts/setup-mega-app.sh -F react-native",
     "setup:react-native:cli": "./scripts/setup-mega-app.sh -F react-native -B cli",
     "setup:react-native:expo": "./scripts/setup-mega-app.sh -F react-native -B expo",
-    "react-latest-cra-latest-ts": "npm run setup:react:cra -- -P yarn",
-    "react-latest-next-latest-ts": "npm run setup:react:next",
-    "react-latest-vite-latest-ts": "npm run setup:react:vite",
-    "react-16-cra-latest-ts": "npm run setup:react:cra -- -f 16",
-    "react-latest-cra-latest-js": "npm run setup:react:cra -- -l js",
-    "react-18-next-12-ts": "npm run setup:react:next -- -f 18 -b 12",
-    "react-latest-vite-2-ts": "npm run setup:react:vite -- -b 2",
-    "angular-latest-angular-cli-latest-ts": "npm run setup:angular:cli",
-    "angular-14-angular-cli-14-ts": "npm run setup:angular:cli -- -f 14 -b 14 -n angular-14-angular-cli-14-ts",
-    "vue-3-vue-cli-latest-ts": "npm run setup:vue:cli -- -f 3 -P yarn",
-    "vue-latest-vite-latest-ts": "npm run setup:vue:vite",
-    "vue-latest-vite-3-ts": "npm run setup:vue:vite -- -b 3",
-    "vue-latest-nuxt-latest-ts": "npm run setup:vue:nuxt",
-    "rnlatestclilatesttsios": "npm run setup:react-native:cli -- -A ios -n rnLatestCliLatestTsIos",
-    "rnlatestclilatesttsandroid": "npm run setup:react-native:cli -- -A android -n rnLatestCliLatestTsAndroid",
-    "rnlatestexpolatesttsandroid": "npm run setup:react-native:expo -- -A android -n rnLatestExpoLatestTsAndroid",
+    "react-latest-cra-latest-ts-ui-latest": "npm run setup:react:cra -- -P yarn",
```

Comment:
Adding a Amplify UI tag to each test, `latest` is used by default.

Definitely open to ideas for improving these tests but I just followed the existing structure for now so we can unblock the pipeline, and can do some more rethinking when we merge the v1 (canary apps) and v2 (mega-apps) build tests.

Reply:
Nit: I know you didn't set all this up but can we make a follow up to parameterize the the framework and the dist tag? Seems like it's pretty annoying here to have to update the command on every single line

Reply:
Yeah I'll add that as an AC on the followup task

Reply:
+1